### ISSUE TITLE
Is it possible to replicate the location behavior of the NYC Subway app?

### ISSUE THREAD

**Issue by samandmoore** [tidbyt/community#110]
Hi there! 

I'm really excited about this new community apps approach. Thanks for doing this. 

## TL;DR
I'm wondering if it's possible to achieve the same thing that the existing NYC Subway departures app does with the location schema? I need a way to accept a user's location and then use that to generate a list of bus steps nearby so that the user can choose one for the app.

## Context
I've built a [NYC MTA Bus departures app](https://github.com/samandmoore/pixlet-mta-bus-time) and it's working great for me in my home setup today (currently I'm rendering it minutely in AWS Lambda 🤪).

I'd like to contribute it back to the community. I modeled it off of the built-in NYC MTA Subway departures app, but obviously since that's closed source, it's not quite the same. 

In the app today, I've just hard-coded it to use a stop that I care about. In order to make the stop dynamic, I'd need to be able to access the user's location and then query the MTA API to provide stops nearby. This appears to be exactly what the existing Subway departures app does, but I can't quite figure out how to replicate that. It seems like it's not possible.

**Comment by samandmoore** [comment]
this was resolved with the addition of the `LocationBased` schema type. thanks!
